Catalog/src/WebApi/WebApi.csproj

```diff
@@ -0,0 +1,40 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <RootNamespace>Catalog.WebApi</RootNamespace>
+        <AssemblyName>Catalog.WebApi</AssemblyName>
+        <Nullable>enable</Nullable>
+        <IsPackable>true</IsPackable>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <UserSecretsId>efad71c6-743c-4b87-9de8-f26d77146f6d</UserSecretsId>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="..\Application\Application.csproj" />
+        <ProjectReference Include="..\Infrastructure\Infrastructure.csproj" />
```

Comment:
The dependency is only required because the applicationm exceptions are referenced here:
https://github.com/jozsef-halmi/D3plus/blob/feature/setup-catalog-service/Catalog/src/WebApi/Filters/ApiExceptionFilterAttribute.cs#L17